pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -238,6 +238,17 @@ public String alterTableStateOrListTableConfig(
   public SuccessResponse deleteTable(
       @ApiParam(value = "Name of the table to delete", required = true) @PathParam("tableName") String tableName,
       @ApiParam(value = "realtime|offline", required = false) @QueryParam("type") String tableTypeStr) {
+    TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableName);
```

Comment:
We have generally followed the convention of specifying a 'type', keeping the suffix _REALTIME and _OFFLINE internal to pinot. Granted there are a couple of APIs that use the suffix, but let us not propagate it (unless there is a specific issue).